Makefile

```diff
@@ -78,6 +79,15 @@ $(warning No code-sign identity found (CODE_SIGN_IDENTITY is not set))
 endif
 endif
 
+ifeq ($(TARGET_PLATFORM),win32)
+ifndef CODE_SIGN_CERTIFICATE
+$(warning No code-sign certificate found (CODE_SIGN_CERTIFICATE is not set))
+endif
+ifndef CODE_SIGN_CERTIFICATE_PASSWORD
+$(warning No code-sign certificate password found (CODE_SIGN_CERTIFICATE_PASSWORD is not set))
+endif
```

Comment:
And for those scripts where you have the option to verify the signature, maybe it would make sense to sign the package / exe in the temporary directory, and only move it to `release/out/signed/` after it's been verified.